mlflow/gateway/providers/openai.py

```diff
@@ -189,7 +188,7 @@ def sync_completions(self, payload: completions.RequestPayload) -> completions.R
 
         # use python requests instead of aiohttp
         resp = requests.post(
-            url=os.path.join(self._request_base_url, "chat/completions"),
+            url=self._request_base_url + "/chat/completions",
```

Comment:
Can we use urlparse to parse the base URL, then do a posixpath.join(), then reconstruct using urlunparse? I think we already have utils in MLflow for this (append_to_uri_path)